md/database-configuration-for-business-data.md

```diff
@@ -18,7 +18,7 @@ Configuration is done per tenant. If you have multiple tenants to configure, you
 
 ## Database creation
 
-In your RDBMS, create a database to be used to store business object. Do not reuse the Bonita BPM Engine database.
+In your RDBMS, create a database to be used to store business object. Do not reuse the Bonita BPM Engine database. Follow the instructions in the [Specific database configuration](database-configuration.md#Specific database configuration) section.
```

Comment:
Have a look at https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md for how to use anchors